internal/jimm/juju/juju.go

```diff
@@ -14,9 +14,8 @@ import (
 	"github.com/canonical/jimm/v3/internal/openfga"
 )
 
-// JIMM is a placeholder name for jujuManager.
-// This struct manages all business logic with Juju resources.
-type JIMM struct {
+// JujuManager handles all business logic with Juju resources.
+type JujuManager struct {
```

Comment:
we could simply call it `Manager` since it is in the juju package..

Reply:
True, but I'll leave that for a separate PR.